frontend/packages/dev-console/integration-tests/views/serverless.view.ts

```diff
@@ -0,0 +1,42 @@
+import { browser, ExpectedConditions as until, by, element } from 'protractor';
+import * as sidenavView from '../../../../integration-tests/views/sidenav.view';
+import * as crudView from '../../../../integration-tests/views/crud.view';
+import { switchPerspective, Perspective, sideHeader } from './dev-perspective.view';
+
+export const firstKebabMenu = element.all(by.css('[data-test-id="kebab-button"]')).first();
+export const actionItems = element(by.css('data-test-id="action-items"'));
+export const editCount = element.all(by.css('.pf-c-dropdown__menu-item')).first();
+export const machineSetsInput = element(
+  by.className('pf-c-form-control co-m-number-spinner__input'),
+);
+export const submitCount = element(by.id('confirm-action'));
+export const cellClass = element(by.cssContainingText('td.col-sm-4.hidden-xs', '9 of 9 machines'));
+export const pageSidebar = element(by.id('page-sidebar'));
+export const installServerlessOperator = element(
+  by.css('div.pf-c-modal-box.co-catalog-page__overlay.co-catalog-page__overlay--right'),
+);
+export const createProject = element(by.id('yaml-create'));
+export const nameofProject = element(by.id('input-name'));
+export const displayNameofProject = element(by.id('input-display-name'));
+
+// Creating namespace from UI
+export const createNewProject = async (projectName: [string, string]) => {
+  await browser.wait(until.presenceOf(sidenavView.navSectionFor('Home')));
+  await sidenavView.clickNavLink(['Home', 'Projects']);
+  await crudView.isLoaded();
+  await browser.wait(until.presenceOf(createProject));
+  await createProject.click();
+  await browser.wait(until.presenceOf(nameofProject));
+  await nameofProject.sendKeys(projectName[0]);
+  await displayNameofProject.sendKeys(projectName[1]);
+  await submitCount.click();
+};
```

Comment:
above you have done `await execSync(`oc new-project knative-serving`);`  https://github.com/openshift/console/pull/4136/files#diff-6867656b69ac7e61e3433dac07ff7b9fR110

so this seems unused to me

Reply:
It was created for the future when we are going to create new serverless application and it will be used. The oc command will reduce the execution time and the main purpose of this automation is installation of serverless operator. That's why it isn't used in this PR.

Reply:
i don't think we should have unused code , we can add later when needed

Reply:
Okay. I will remove this part.